images/linux/scripts/installers/7-zip.sh

```diff
@@ -4,6 +4,8 @@
 ##  Desc:  Installs 7-zip
 ################################################################################
 
+set -e
+set -o pipefail
```

Comment:
Does `packer` provides any way to set default `erroraction` behavior for all script on template level?
I worry that we can just forget to add this line to new script so it would be great to apply this behavior for all scripts without putting code to every script

Reply:
It looks like the packer checks only the last exit code from the script, but if something went wrong in the middle we can't see it. I've faced with a lot of such cases during macOS provision debugging

Reply:
Let's investigate another ways to do it. Including `set -e` to every script looks like bad solution here.
It will require additional maintenance and supporting. We will have to validate manually that all future scripts will contain this line.
Ideally, we need something like default error action parameter for all image provisioners

Reply:
@maxim-lobanov we should either get rid of` /bin/bash` at all or put `-e` judging by the packer docks
![image](https://user-images.githubusercontent.com/48208649/94806676-088a0300-03f7-11eb-8f4e-514e9c17bb9e.png)

Reply:
However, getting rid of `/bin/bash` doesn't help me

Reply:
For Windows, I saw solution: `Add-Content -Path $PSHOME/Profile.ps1 -Value "$ErrorActionPreference='Stop'"`